wallet/txauthor/author.go

```diff
@@ -95,7 +95,7 @@ func NewUnsignedTransaction(outputs []*wire.TxOut, feeRatePerKb btcutil.Amount,
 
 	targetAmount := SumOutputValues(outputs)
 	estimatedSize := txsizes.EstimateVirtualSize(
-		0, 1, 0, outputs, changeSource.ScriptSize,
+		0, 0, 1, 0, outputs, changeSource.ScriptSize,
```

Comment:
hmm, unfortunately this change won't take effect until we push yet another version of the `txauthor` package.

Reply:
Ah true, since we updated `txauthor` in the past PR that added this to `txsizes`...